tiledb/py_arrow_io_impl.h

```diff
@@ -199,6 +199,10 @@ ArrowInfo tiledb_buffer_arrow_fmt(BufferInfo bufferinfo, bool use_list = true) {
   case TILEDB_FLOAT64:
     return ArrowInfo("g");
   case TILEDB_BLOB:
+#if TILEDB_VERSION_MAJOR >= 2 && TILEDB_VERSION_MINOR >= 21
```

Comment:
if there's a tiledb 3 some day:
```suggestion
#if TILEDB_VERSION_MAJOR > 2 || (TILEDB_VERSION_MAJOR == 2 && TILEDB_VERSION_MINOR >= 21)
```

Reply:
Nice observation. It's acceptable for now, but it will change throughout the entire project in the future.